stylesheets/commons/FilterButtons.less

```diff
@@ -58,6 +58,52 @@ Author(s): Elysienna
 	animation-duration: 0.8s;
 }
 
+.dota2-test {
+	.filter-buttons {
+		display: flex;
+		justify-content: start;
+		flex-flow: row wrap;
+		font-size: 10pt;
+		margin: 0.125rem;
+		align-items: center;
+	}
+
+	.filter-button {
+		display: flex;
+		align-items: center;
+		border-radius: 8px;
+		padding: 6px 12px;
+		font-weight: bold;
+		flex-grow: initial;
+
+		.theme--light & {
+			color: var( --clr-on-primary-container, #ffffff );
+			background-color: var( --clr-primary-container, #1f3d7a );
+			border-color: var( --clr-primary-container, #1f3d7a );
+		}
+
+		.theme--dark & {
+			color: #ffffff;
+			background-color: var( --clr-surface-3, #282828 );
+			border-color: var( --clr-surface-3, #282828 );
+		}
+	}
+
+	.filter-button--active {
+		.theme--light & {
+			color: #ffffff;
+			background-color: var( --clr-wiki-primary, #1f3d7a );
+			border-color: var( --clr-wiki-primary, #1f3d7a );
+		}
+
+		.theme--dark & {
+			color: #ffffff;
+			background-color: var( --clr-wiki-theme-primary, #1f3d7a );
+			border-color: var( --clr-wiki-theme-primary, #1f3d7a );
+		}
```

Comment:
You need to test this first, but I think we should use `--clr-wiki-theme-primary` for both light and dark theme. That new variable has the correct colors that match to the new design. That variable is also used in the new panel box headers.

Reply:
Updated